### PR TITLE
Examples

### LINKED ISSUE #24: [JOSS 6532] Case Study
You do already have the example primitive written in the paper. However, I think it would be more compelling and/or easier to use if you also included an end-to-end example demonstrating an application which required the use of jaxbind. For example binding to a python odesolver [which calls c++ under the hood].

I think this would be quite valuable for users to see how they might apply it to their application (beyond the abstract evaluation individual description in the picture).

For now I'm going to wait on marking the checkbox 
" Example usage: Do the authors include examples of how to use the software (ideally to solve real-world analysis problems)." pending a response here.

ref: https://github.com/openjournals/joss-reviews/issues/6532

### PR DESCRIPTION
**Pull request by roth-jakob**
add comments to jaxducc and export to the webpage

### CODE REVIEW

File: jaxbind/contrib/jaxducc0.py [hunk 1]
```diff
@@ -53,11 +94,26 @@ def _fht_abstract(*args, **kwargs):
     return ((x.shape, x.dtype, out_ax),)
 
 
+# %% [markdown]
+
+# How we register the hartly transformation JAX primitive via the
```
Comment (Edenhofer) [inline jaxbind/contrib/jaxducc0.py]: How -> Now

File: jaxbind/contrib/jaxducc0.py [hunk 2]
```diff
@@ -30,6 +50,20 @@ def _realtype(dtype):
...
 
+# %% [markdown]
+
+# ## Binding the DUCC hartly transform to JAX
+
+# In the following we provide a Python `_fht` calling C++ hartly transformation
+# of DUCC ` ducc0.fft.genuine_fht`. The C++ function natively supports
+# batching along axis. To make use of that the the python function also
+# translates potential batching axis of JAX to axis of DUCC. Detailed
+# explanations on custom batching along axis can be found in the
```
Comment (Edenhofer) [inline jaxbind/contrib/jaxducc0.py:61]: axis should probably be either "an axis" or "multiple axes"

File: docs/generate.sh [hunk 3]
```diff
@@ -2,12 +2,16 @@
...
 
-FOLDER=docs/source/demos/
+FOLDER=docs/source/demos
 
 for file in "${FOLDER}/01_linear_function" "${FOLDER}/02_multilinear_function" "${FOLDER}/03_nonlinear_function"; do
 	jupytext --to ipynb "${file}.py"
     jupyter-nbconvert --to markdown --execute --ExecutePreprocessor.timeout=None "${file}.ipynb"
 done
 
+FOLDER=docs/source/jaxducc
```
Comment (Edenhofer) [inline docs/generate.sh]: * Why redefine `FOLDER` here? If we want to redefine it, I would make it a lower-case variable 
* Isn't the true folder rather "docs/source/contrib"?

File: docs/generate.sh [hunk 4]
```diff
@@ -2,12 +2,16 @@
...
+jupytext --to ipynb "docs/source/contrib/jaxducc0.py"
```
Comment (Edenhofer) [inline docs/generate.sh:12]: I think you forgot to link `jaxducc0` into "docs/source/contrib". As is, the documentation does not build for me.

**Review by Edenhofer — COMMENTED** [review]
Comments look very good! I think we are still missing a symlink to the `jaxducc0` file in the docs folder but otherwise this looks good. A couple of minor comments are below (above?).

File: jaxbind/contrib/jaxducc0.py (hunk 1 above)
Comment (roth-jakob) [reply jaxbind/contrib/jaxducc0.py]: Thanks

File: jaxbind/contrib/jaxducc0.py (hunk 2 above)
Comment (roth-jakob) [reply jaxbind/contrib/jaxducc0.py:61]: Thanks

File: docs/generate.sh (hunk 3 above)
Comment (roth-jakob) [reply docs/generate.sh]: ups, yes the FOLDER variable is not needed. It's a leftover before I set the full path for the jaxducc file

File: docs/generate.sh (hunk 4 above)
Comment (roth-jakob) [reply docs/generate.sh:12]: Ah thanks, I had not added the docs/sources/contrib folder to git before pushing

**Pull request merged**